App.tsx

```diff
@@ -24,6 +24,7 @@ import {
   LearnMoreLinks,
   ReloadInstructions,
 } from 'react-native/Libraries/NewAppScreen';
+import SignUpScreen from './Source/UI/Screens/SignUpScreen';
```

Comment:
This is imported but not used, can we remove it

Reply:
Yes sir i have removed it.